frontend/templates/views/partials/video.j2

```diff
@@ -1,4 +1,6 @@
 {% do doc.styles.addCssFile('css/components/atoms/video.css') %}
+{% do doc.amp_dependencies.add('amp-video', '0.1') %}
+{% do doc.amp_dependencies.add('amp-youtube', '0.1') %}
```

Comment:
Only one of `amp-video` or `amp-youtube` is needed, put them in the corresponding if conditions.